src/plugins/twitter/twitter-fr.hbs

```diff
@@ -20,18 +20,39 @@
 <div lang="en">
 <p><strong>Needs translation</strong></p>
 <section>
-	<h2>Example: Embedded User Timeline</h2>
-	<div class="wb-twitter">
-		<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets by @WebExpToolkit</a>
-	</div>
+	<h2>Examples: Profile timeline</h2>
+
+	<div class="row">
+		<section class="col-md-6">
+			<h3>Normal account</h3>
+			<div class="wb-twitter">
+				<a class="twitter-timeline" href="https://twitter.com/WebExpToolkit" data-tweet-limit="3">Tweets by @WebExpToolkit</a>
+			</div>
+			<section>
+				<h4>Code</h4>
+				<pre><code>&lt;div class=&quot;wb-twitter&quot;&gt;
+	&lt;a class=&quot;twitter-timeline&quot; href=&quot;https://twitter.com/WebExpToolkit&quot; data-tweet-limit=&quot;3&quot; &gt;Tweets by @WebExpToolkit&lt;/a&gt;
+&lt;/div&gt;</code></pre>
+			</section>
+		</section>
 
-	<section>
-		<h3>Code</h3>
-		<pre><code>&lt;div class=&quot;wb-twitter&quot;&gt;
-&lt;a class=&quot;twitter-timeline&quot; href=&quot;https://twitter.com/WebExpToolkit&quot; data-tweet-limit=&quot;3&quot; &gt;Tweets by @WebExpToolkit&lt;/a&gt;
+		<section class="col-md-6">
+			<h3>"Verified" account</h3>
+			<div class="wb-twitter">
+				<a class="twitter-timeline" href="https://twitter.com/Canada" data-tweet-limit="3">Tweets by @Canada</a>
```

Comment:
```suggestion
				<a class="twitter-timeline" href="https://twitter.com/Canada" data-tweet-limit="3" data-lang="fr">Tweets by @Canada</a>
```

Reply:
Skipping/marking as resolved due to https://github.com/wet-boew/wet-boew/pull/9675#discussion_r1375055677.